content/api/plugins/writing-a-plugin.md

```diff
@@ -31,9 +31,9 @@ The plugins file must export a function with the following signature:
 
 <Alert type="warning">
```

Comment:
Other than updating this `Alert` per my previous comment, can you revert all changes in the "writing a plugin" file? This file will need to be rewritten as part of UNIFY-321.